opdracht1.js

```diff
@@ -0,0 +1,38 @@
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+let allTvNames = [];
+inventory.map(function (value){
+    allTvNames.push(value.type);
+});
+
+console.log(allTvNames);
+
+// Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+let soldOutTvs = [];
+inventory.map(function (value){
+    if (value.originalStock === value.sold){
+        soldOutTvs.push(value);
+    }
+});
```

Comment:
De map gebruik je hier als een forEach. Beter is het om een filter te gebruiken:
```suggestion
inventory.filter(value => value.originalStock === value.sold);
```